mlir/test/Dialect/Linalg/generalize-named-ops.mlir

```diff
@@ -996,9 +994,198 @@ func.func @matmul_transpose_a_b_explicit(%arg0: memref<5x3xf32>, %arg1: memref<7
                       ]
                       ins(%arg0, %arg1 : memref<5x3xf32>, memref<7x5xf32>)
                       outs(%arg2: memref<3x7xf32>)
-                      
   return
 }
 
 // -----
 
+// CHECK: #[[$ATTR_0:.+]] = affine_map<(d0, d1, d2) -> (d0, d2)>
+// CHECK: #[[$ATTR_1:.+]] = affine_map<(d0, d1, d2) -> (d2, d1)>
+// CHECK: #[[$ATTR_2:.+]] = affine_map<(d0, d1, d2) -> (d0, d1)>
+
+// CHECK-LABEL:   func.func @contract_matmul(
+// CHECK-SAME:                                             %[[VAL_0:.*]]: memref<3x5xf32>,
+// CHECK-SAME:                                             %[[VAL_1:.*]]: memref<5x7xf32>,
+// CHECK-SAME:                                             %[[VAL_2:.*]]: memref<3x7xf32>) {
```

Comment:
[nit] Could you reduce the level of invention here? It's actually quite tricky to read/review on my tablet 😅 But in general, shorter lines are better.

Reply:
As above - I hope I got them al! (Note that for the most part I was keeping to the style of the surrounding tests though 😬 )

Reply:
Thanks for the update.

> Note that for the most part I was keeping to the style of the surrounding tests though 😬 )

Yes, consistency is key :) 

TODO for myself: unify the format in this file.